tests/console/commands/test_update.py

```diff
@@ -80,3 +83,21 @@ def test_update_prints_operations(
 
     assert ("Package operations:" in output) is expected
     assert ("Installing docker (4.3.1)" in output) is expected
+
+
+@pytest.fixture()
```

Comment:
```suggestion
```

not a fixture but a test

Reply:
Ah. Sorry, misunderstood, thought this was injecting the fixtures. New to pytest.